environment.yml

```diff
@@ -3,72 +3,77 @@ channels:
   - defaults
 
 dependencies:
-  - kalasiris 
-  - ale=0.9.1
+  - ale>=0.9.1, <1
   - armadillo
-  - boost=1.72
-  - boost-cpp=1.72
+  - boost>=1.72.0, <2
+  - boost-cpp>=1.72.0, <2
   - blas
   - usgs-astrogeology::bullet
   - bz2file
   - bzip2
-  - cmake >=3.15
+  - cmake >=3.15, <4
   - cspice=67
   - csm>=3.0.3,<3.0.4
-  - curl
-  - cxx-compiler=1.1.2
+  - curl>=8.3.0,<9.0a0
+  - cxx-compiler>=1.1.2
   - doxygen
-  - eigen < 3.4
-  - embree>=2.17,<3
+  - eigen>=3, <4
+  - embree>=2.17, <3
   - ffmpeg
-  - geos =3.11.1
+  - geos>=3.11
```

Comment:
Is it worth capping the version so that we don't run into build errors with future releases?